g3doc/user_guide/gpu.md

```diff
@@ -115,9 +115,29 @@ As a result, `nvproxy` has the following limitations:
 
 ### Supported GPUs {#gpu-models}
 
-gVisor currently supports NVIDIA GPUs: T4, L4, A100, A10G and H100. Please
```

Comment:
A10G GPU is removed from the list, will it be officially supported?

Reply:
It's not removed, it's grouped with the A100 bullet point.